test/e2e/network/dual_stack.go

```diff
@@ -367,7 +367,7 @@ var _ = common.SIGDescribe("[Feature:IPv6DualStack]", func() {
 		expectedPolicy := v1.IPFamilyPolicyRequireDualStack
 		expectedFamilies := []v1.IPFamily{v1.IPv4Protocol, v1.IPv6Protocol}
 
-		service := createService(t.ServiceName, t.Namespace, t.Labels, nil, expectedFamilies)
```

Comment:
these tests only run on dual-stack clusters, they are not running automatically in presubmits, that is why we didn't hit it.
They run periodically though, we should be able to catch it later

Reply:
Aha!